common/src/stack/command/stack/commands/__init__.py

```diff
@@ -1893,49 +1893,49 @@ def endOutput(self, header=[], padChar='-', trimOwner=False, trimHeader=False):
 				for i in range(0, rows):
 					header.append('col-%d' % i)
 
-			list = []
+			output_lines = []
 			for line in self.output:
-				dict = {}
+				dictionary = {}
```

Comment:
If you're going to lint, you might as well make the variable name a little more descriptive while you're at it.

Reply:
+1

Reply:
I couldn't figure out a good name in the moment, so suggestions would be appreciated.